docs/tutorials/image_signing.md

```diff
@@ -0,0 +1,189 @@
+# How to sign and distribute container images using Podman
+
+Signing container images originates from the motivation of trusting only
+dedicated image providers to mitigate man-in-the-middle (MITM) attacks. One way
+to sign images is to utilize a GNU Privacy Guard ([GPG][0]) key. This technique
+is generally compatible with any OCI compliant container registry like
+[Quay.io][1]. It is worth mentioning that the OpenShift integrated container
+registry supports this signing mechanism out of the box.
+
+[0]: https://gnupg.org
+[1]: https://quay.io
+
+From a technical perspective, we can utilize Podman to sign the image before
+pushing it into a remote registry. After that, all systems running Podman have
+to be configured to retrieve the signature key from a remote server, which can
+be any simple web server. This means that every unsigned image will be rejected
+during an image pull operation. But how does this work?
+
+First of all, we have to create a GPG key pair or select an already locally
+available one:
+
+```bash
+> gpg --list-keys sgrunert@suse.com
+pub   rsa2048 2018-11-26 [SC] [expires: 2020-11-25]
+      XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+uid           [ultimate] Sascha Grunert <sgrunert@suse.com>
+sub   rsa2048 2018-11-26 [E] [expires: 2020-11-25]
+```
+
+Now let’s assume that we run a container registry. For example we could simply
+start one on our local machine:
+
+```bash
+> sudo podman run -d -p 5000:5000 registry
+```
+
+The registry does not know anything about image signing, it just provides the remote
+storage for the container images. This means if we want to sign an image, we
+have to take care of how to distribute the GPG keys.
```

Comment:
… the signatures?

(Yes, you also have to distribute the public keys to consumers so that their `policy.json` files can use them. But most of the blog deals with the sigstore mechanism for distributing signatures.)

Reply:
Yep, changed it to `signatures`.